FhirBlaze.QuestionnaireModule/Pages/EditQuestionnaire.razor.cs

```diff
@@ -0,0 +1,70 @@
+﻿using FhirBlaze.SharedComponents.Services;
+using Hl7.Fhir.Model;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using Task = System.Threading.Tasks.Task;
+using static Hl7.Fhir.Model.Questionnaire;
+
+namespace FhirBlaze.QuestionnaireModule.Pages
+{
+    [Authorize]
+    public partial class EditQuestionnaire
+    {
+        [Inject]
+        public IFhirService FhirService { get; set; }
+        [Inject]
+        public HttpClient Http { get; set; }
+        [Inject]
+        public NavigationManager NavigationManager { get; set; }
+        [Parameter]
+        public string Id { get; set; }
+        public bool QLoaded { get; set; } = false;
+        public Questionnaire Questionnaire { get; set; } = new Questionnaire();
+
+        protected override async Task OnInitializedAsync()
+        {
+            try
+            {
+                Questionnaire = await FhirService.GetQuestionnaireByIdAsync(Id);
+                QLoaded = true;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.ToString());
+            }
+        }       
+
+        protected void AddItem(QuestionnaireItemType type)
```

Comment:
This method is not used; please remove

Reply:
This was used before I added the questionnaire component.